app/services/ccms/requestors/base_requestor.rb

```diff
@@ -46,22 +46,6 @@ def transaction_request_id
 
     private
 
-      def soap_client
-        @soap_client ||= Savon.client(
-          headers: { "x-api-key" => config.aws_gateway_api_key },
-          env_namespace: :soap,
-          wsdl: wsdl_location,
-          namespaces:,
-          pretty_print_xml: true,
-          convert_request_keys_to: :none,
-          namespace_identifier: "ns2",
-          log: false,
-          log_level: :debug,
-          open_timeout: 180,
```

Comment:
@colinbruce had a thought after approving this pr, we are removing the timeout here, have we tested how the timeouts are handled with Faraday?

Reply:
I _think_ they should be handled by the SoapCall class and passed back as Faraday::SoapErrors.

Let me do some testing 👍